deployment/kubernetes/overlays/scanner/config/preflight.yaml

```diff
@@ -1,5 +1,6 @@
 # this cluster-name is just used to mark the generated reports
 cluster-name: mycluster
+bucket-name: mybucket
```

Comment:
I don't think you can specify a bucket here.

Reply:
ooops, that is mistake.